scripts/bundleSizeConfig.js

```diff
@@ -1,6 +1,6 @@
 module.exports = {
   // Size limit for all bundles used by each service (K)
   // Keep these +/- 5K and update frequently!
-  MIN_SIZE: 769,
-  MAX_SIZE: 813,
+  MIN_SIZE: 784,
+  MAX_SIZE: 828,
```

Comment:
This is fairly big for something that only affects the test environment

Is it the `url-parse` package?  If so, that seems overkill, unless I'm missing something

Something as simple as `pathname.concat(isLive() ? '' : '?renderer_env=live')` would be fine for me

Reply:
Interestingly, it seems to be the URLSeachParams that @jroebu14 mentioned in the previous comment. Webpack seems to be using polyfills to ensure that that function is available across browsers.

I used url-parse because I needed to ensure that when `renderer_env=live` or any other query param is used, the final url sent to `fetchPageData` is still good.

The current  jenkins build is failing because bundle size is too small after I removed URLSearchParams so I'll be undoing my changes to the bundleSizeConfig